homepage/templates/homepage/add_review_search.html

```diff
@@ -0,0 +1,24 @@
+<html>
+<head>
+<title>Class Rater Add Review Search</title>
+</head>
+
+<body>
+{% extends 'homepage/app_layout.html' %}
+{% block content %}
+<div class="container-fluid text-left pt-5">
+  <div class="row">
+    <div class="col-sm-4">    
+        {% if courses %}
+        <h2>Click on Course to start a review</h2>
+        {% for course in courses %}
+            <a href="{% url 'add_review' %}">{{ course }}</a></br>

```

Comment:
Does this reroute to the page with the data already (filled in course name and professors possibilities) or is that up to further development?

Reply:
At the moment, it reroutes to the general **add review** page without the data.
In the next PR, I'll update the **add review** page and modify that as you described.